net-misc/i2pd/i2pd-2.6.0-r3.ebuild

```diff
@@ -53,27 +53,38 @@ src_configure() {
 
 src_install() {
 	cmake-utils_src_install
+
+	# config
+	insinto "/etc/i2pd"
+	doins "${S}/docs/i2pd.conf"
+	doins "${S}/debian/subscriptions.txt"
+	doins "${S}/debian/tunnels.conf"
+
+	# doc
 	dodoc README.md
+
+	# working directory
 	keepdir /var/lib/i2pd/
-	insinto "/var/lib/i2pd"
+	insinto /var/lib/i2pd
 	doins -r "${S}/contrib/certificates"
 	dosym /etc/i2pd/subscriptions.txt /var/lib/i2pd/subscriptions.txt
 	fowners "${I2PD_USER}:${I2PD_GROUP}" /var/lib/i2pd/
 	fperms 700 /var/lib/i2pd/
-	dodir "/etc/${PN}"
-	insinto "/etc/${PN}"
-	doins "${S}/docs/${PN}.conf"
-	doins "${S}/debian/subscriptions.txt"
-	doins "${S}/debian/tunnels.conf"
```

Comment:
In this commit I did not move any files, I just regrouped the lines in the install section. But yes, `i2pd.conf`, `tunnels.conf` and `subscriptions.txt` are the config files that users may want to change. It is our choice where to place them, and for a long time we are placing them in `/etc/i2pd`. This is not changing here, and our current openrc and systemd scripts expect to find these files in `etc/i2pd`.